cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/SourceControlManagementServiceTest.java

```diff
@@ -80,7 +77,12 @@ public void testSetRepoConfig() throws Exception {
 
     //clean up
     namespaceAdmin.delete(namespaceId);
-    Assert.assertNull(sourceControlService.getRepository(namespaceId));
+
+    try {
+      sourceControlService.getRepository(namespaceId);
```

Comment:
need an Assert.fail() after this line (within the try block). Otherwise this test will pass regardless of whether the exception is thrown or not

Reply:
Added Assert.fail() in these exception expectant try blocks